testapi.pm

```diff
@@ -99,11 +99,11 @@ sub init {
         $serialdev = "hvc0";
     }
     elsif (check_var('VIRSH_VMM_FAMILY', 'xen') && check_var('VIRSH_VMM_TYPE', 'linux')) {
-        if (check_var('VERSION', '12-SP2')) {
-            $serialdev = "hvc0";
+        if (check_var('VERSION', '12-SP1')) {
+            $serialdev = "xvc0";
         }
         else {
-            $serialdev = "xvc0";
+            $serialdev = "hvc0";
```

Comment:
Actually I don't quite get why we need to distinguish between SLE versions in the testapi. That is pretty wrong. Maybe you want to consult with @mnowaksuse and come up with an approach where the backend does not need to know anything about SLE. Maybe some `$distri` method can help here, e.g. as is done for `script_run` and such.